pkg/app/app.go

```diff
@@ -102,6 +102,8 @@ func NewApp(config config.AppConfigurer, filterPath string) (*App, error) {
 	}
 	var err error
 	app.Log = newLogger(config)
+
+	i18n.UserConfigLanguage = config.GetUserConfig().Gui.Language
```

Comment:
rather than introduce a global variable that we mutate, which can lead to problems down the line, what are your thoughts on passing `config.GetUserConfig().Gui.Language` as an argument to `NewTranslationSet` instead?